spec/controllers/search_controller_spec.rb

```diff
@@ -47,27 +47,27 @@
       end
     end
 
-    context 'with invalid UTF-8' do
-      let(:query) { "\x99" }
+    # context 'with invalid UTF-8' do
+    #   let(:query) { "\x99" }
 
-      it 'searches for an empty string' do
-        expect(GroupSearch).to receive(:new).with('', an_instance_of(SearchResults)).and_return(group_search)
-        expect(PersonSearch).to receive(:new).with('', an_instance_of(SearchResults)).and_return(person_search)
-        subject
-      end
+    #   it 'searches for an empty string' do
+    #     expect(GroupSearch).to receive(:new).with('', an_instance_of(SearchResults)).and_return(group_search)
+    #     expect(PersonSearch).to receive(:new).with('', an_instance_of(SearchResults)).and_return(person_search)
+    #     subject
+    #   end
 
-      it 'assigns the search result to @people_results' do
-        subject
-        expect(assigns(:people_results)).to eq(people_results)
-      end
+    #   it 'assigns the search result to @people_results' do
+    #     subject
+    #     expect(assigns(:people_results)).to eq(people_results)
+    #   end
 
-      it 'assigns team search result to @team_results' do
-        subject
-        expect(assigns(:team_results)).to eq(team_results)
-      end
-    end
+    #   it 'assigns team search result to @team_results' do
+    #     subject
+    #     expect(assigns(:team_results)).to eq(team_results)
+    #   end
+    # end
```

Comment:
Maybe let's add a TODO for the commented out spec and create a tech debt ticket to return to it subsequently? I think you suggested maybe we'd have another look at this after the 5.0->5.2 and if it doesn't work then, maybe we can consider deleting the spec at that point.

Reply:
Absolutely, sounds like a good approach. I'll add a TODO / ticket in the next PR 👍